modules/manage/pages/cluster-maintenance/compaction-tuning.adoc

```diff
@@ -0,0 +1,48 @@
+= Optimize Compaction Settings
+:description: Redpanda's approach to compaction and options for configuring it
+
+Configure compaction for your cluster to optimize storage utilization.
+
+== Repanda Compaction Overview
+
+Compaction is an optional mechanism intended to reduce the storage needs of Redpanda topics. Compaction is enabled through configuration of a cluster or topic's cleanup policy. When compaction is enabled as part the cleanup policy, a background process executes on a pre-set interval to perform compaction operations. When triggered for a partition the process purges older versions of messages for a given key and only retains the most chronologically recent message in that partition. This is done by analyzing closed segments in the partition, copying the most recent messages for each key into a new segment,then deleting the source segments.
```

Comment:
> chronologically recent 

redundant? not sure

Reply:
Logs are FIFO ordered--like physically in the order they arrive. There are also timestamps attached, but I'm pretty sure that the timestamps do not ever affect the ordering in the log. In practice I think time-based ordering is desired, but I doubt it is enforced.